client/ayon_maya/plugins/publish/extract_pointcache.py

```diff
@@ -516,7 +516,7 @@ def get_attribute_defs(cls):
 
 class ExtractAnimation(ExtractAlembic):
     label = "Extract Animation (Alembic)"
-    families = ["animation"]
+    families = ["animation.abc"]
```

Comment:
By doing this.. do the Extract animation attributes still appear in the publisher Ui?